### PR TITLE
Add Custom registries to Wings 1.0

### PR DESCRIPTION
**Pull request by DomiiBunn**
image

### CODE REVIEW

**Comment by DomiiBunn** [comment]
I am not entirely sure why i changed the whole lock file. Feel free to revert that change. I am unaware of me installing any modules

**Review by parkervcp — DISMISSED** [review]
LGTM

File: wings/1.0/configuration.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Additional Configuration
+
+[[toc]]
+
+::: warning
+These are advanced configurations for the daemon. You risk breaking your daemon and making containers un-usable if
```
Comment (DaneEveritt) [inline wings/1.0/configuration.md:6]: I tend to refer to "the daemon" as "Wings" (proper noun) for the sake of avoiding confusion with 0.6 and for consistency.

**Review by DaneEveritt — CHANGES_REQUESTED** [review]

File: wings/1.0/configuration.md (hunk 1 above)
Comment (DomiiBunn) [reply wings/1.0/configuration.md]: Fixed ^^

**Review by schrej — APPROVED** [review]

**Pull request merged**